db/seeds.rb

```diff
@@ -5,3 +5,47 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+require 'nokogiri'
+require 'open-uri'
+
+Card.destroy_all
+
+page = Nokogiri::HTML(open ("http://www.languagedaily.com/learn-german/vocabulary/common-german-words/"))
+
+page.css('.jsn-article-content > table > tbody > tr').each do |row|
+  de = row.css('td:nth-child(2)')
+  en = row.css('td:nth-child(3)')
+  Card.create(original_text: de.text, translated_text: en.text)
+end
+puts "Loading data from 1 page complite"
+
+page = Nokogiri::HTML(open ("http://www.languagedaily.com/learn-german/vocabulary/most-common-german-words-2"))
+
+page.css('.jsn-article-content > table > tbody > tr').each do |row|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.